modules/security-context-constraints-command-reference.adoc

```diff
@@ -103,35 +104,39 @@ the default SCCs.
 //other than priority, users, groups, labels, and annotations.
 ====
 
-[id="deleting-security-context-constraints_{context}"]
-== Deleting security context constraints
+[id="updating-security-context-constraints_{context}"]
+== Updating security context constraints
 
-To delete an SCC:
+If your custom SCC no longer satisfies your application workloads requirements, you can update your SCC by using the OpenShift CLI (`oc`).
+
+To update an existing SCC:
 
 [source,terminal]
 ----
-$ oc delete scc <scc_name>
+$ oc edit scc <scc_name>
 ----
 
-[NOTE]
+[IMPORTANT]
 ====
-If you delete a default SCC, it will regenerate when you restart the cluster.
+To preserve customized SCCs during upgrades, do not edit settings on
+the default SCCs.
+//other than priority, users, groups, labels, and annotations.
 ====
 
-[id="updating-security-context-constraints_{context}"]
-== Updating security context constraints
+[id="deleting-security-context-constraints_{context}"]
+== Deleting security context constraints
 
-To update an existing SCC:
+If you no longer require your custom SCC, you can delete the SCC by using the OpenShift CLI (`oc`).
+
+To delete an SCC:
 
 [source,terminal]
 ----
-$ oc edit scc <scc_name>
+$ oc delete scc <scc_name>
 ----
 
-[NOTE]
+[IMPORTANT]
 ====
-To preserve customized SCCs during upgrades, do not edit settings on
-the default SCCs.
-//other than priority, users, groups, labels, and annotations.
+Do not delete default SCCs. If you delete a default SCC, it regenerates when you restart the cluster.
```

Comment:
/lgtm
@bergerhoffer PR LGTM. Feel free to update if you like, to be more elaborate:
```suggestion
Do not delete default SCCs. If you delete a default SCC, it is regenerated soon by the Cluster Version Operator.
```
Test:
```
[xxia@2024-08-09 17:34:33 CST my]$ oc delete scc restricted restricted-v2
securitycontextconstraints.security.openshift.io "restricted" deleted
securitycontextconstraints.security.openshift.io "restricted-v2" deleted

[xxia@2024-08-09 17:35:23 CST my]$ oc get scc restricted restricted-v2
NAME            PRIV    CAPS                   SELINUX     RUNASUSER        FSGROUP     SUPGROUP   PRIORITY     READONLYROOTFS   VOLUMES
restricted      false   <no value>             MustRunAs   MustRunAsRange   MustRunAs   RunAsAny   <no value>   false            ["configMap","csi","downwardAPI","emptyDir","ephemeral","persistentVolumeClaim","projected","secret"]
restricted-v2   false   ["NET_BIND_SERVICE"]   MustRunAs   MustRunAsRange   MustRunAs   RunAsAny   <no value>   false            ["configMap","csi","downwardAPI","emptyDir","ephemeral","persistentVolumeClaim","projected","secret"]

[xxia@2024-08-09 17:37:01 CST my]$ oc get clusterversion
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.13.40   True        False         11h     Cluster version is 4.13.40
```

Reply:
Thanks @xingxingxia, I've updated to clarify